source/features/github-actions-indicators.tsx

```diff
@@ -25,7 +33,24 @@ function addTooltip(element: HTMLElement, tooltip: string): void {
 	}
 }
 
-const getWorkflowsDetails = cache.function('workflows', async (): Promise<Record<string, WorkflowDetails> | false> => {
+// Note: There is no way to get a workflow list in the v4 API.
+//       So we get the list through the v3 API and use it as a trusted source.
+//       https://github.com/refined-github/refined-github/pull/6543
+const getWorkflows = async (): Promise<Workflow[]> => {
+	const response = await api.v3(`/repos/${getRepo()!.owner}/${getRepo()!.name}/actions/workflows`);
+
+	const workflows = response.workflows as any[];
+
+	// Note: The response is not reliable.
+	//       Some workflow's path is '' and deleted workflow's state is 'active'.
```

Comment:
When does this happen? Is it just an empty string is or it just a consequence of .split().pop()

Also the state of deleted workflows is consistent/correct. 

```suggestion
```

Reply:
You can check these bugs with calling API of getting refined-github/refined-github's workflow list.
```bash
# GET /repos/refined-github/refined-github/actions/workflows
curl -i -H "Authorization: token YOUR_TOKEN" https://api.github.com/repos/refined-github/refined-github/actions/workflows
```
Here is the query result for saving your time :)
```json
[
    {
        "id": 178729,
        "node_id": "MDg6V29ya2Zsb3cxNzg3Mjk=",
        "name": "Test",
        "path": "",
        "state": "active",
        "created_at": "2020-01-10T06:49:42.000+09:00",
        "updated_at": "2020-01-11T01:03:31.000+09:00",
        "url": "https://api.github.com/repos/refined-github/refined-github/actions/workflows/178729",
        "html_url": "https://github.com/refined-github/refined-github/blob/main",
        "badge_url": "https://github.com/refined-github/refined-github/workflows/Test/badge.svg"
    },
    {
        "id": 49251289,
        "node_id": "W_kwDOAxXxWc4C74PZ",
        "name": "actionlint",
        "path": ".github/workflows/actionlint.yml",
        "state": "active",
        "created_at": "2023-02-23T19:04:16.000+09:00",
        "updated_at": "2023-03-01T18:24:51.000+09:00",
        "url": "https://api.github.com/repos/refined-github/refined-github/actions/workflows/49251289",
        "html_url": "https://github.com/refined-github/refined-github/blob/main/.github/workflows/actionlint.yml",
        "badge_url": "https://github.com/refined-github/refined-github/workflows/actionlint/badge.svg"
    },
    {
        "id": 11449857,
        "node_id": "MDg6V29ya2Zsb3cxMTQ0OTg1Nw==",
        "name": "CodeQL",
        "path": ".github/workflows/codeql-analysis.yml",
        "state": "active",
        "created_at": "2021-07-20T16:08:53.000+09:00",
        "updated_at": "2021-07-20T16:08:53.000+09:00",
        "url": "https://api.github.com/repos/refined-github/refined-github/actions/workflows/11449857",
        "html_url": "https://github.com/refined-github/refined-github/blob/main/.github/workflows/codeql-analysis.yml",
        "badge_url": "https://github.com/refined-github/refined-github/workflows/CodeQL/badge.svg"
    },
    {
        "id": 10572899,
        "node_id": "MDg6V29ya2Zsb3cxMDU3Mjg5OQ==",
        "name": "Features",
        "path": ".github/workflows/features.yml",
        "state": "active",
        "created_at": "2021-06-19T14:56:34.000+09:00",
        "updated_at": "2021-08-24T22:46:39.000+09:00",
        "url": "https://api.github.com/repos/refined-github/refined-github/actions/workflows/10572899",
        "html_url": "https://github.com/refined-github/refined-github/blob/main/.github/workflows/features.yml",
        "badge_url": "https://github.com/refined-github/refined-github/workflows/Features/badge.svg"
    },
    {
        "id": 2844040,
        "node_id": "MDg6V29ya2Zsb3cyODQ0MDQw",
        "name": "Labeler",
        "path": ".github/workflows/labeler.yml",
        "state": "active",
        "created_at": "2020-10-04T14:49:14.000+09:00",
        "updated_at": "2020-10-04T14:49:14.000+09:00",
        "url": "https://api.github.com/repos/refined-github/refined-github/actions/workflows/2844040",
        "html_url": "https://github.com/refined-github/refined-github/blob/main/.github/workflows/labeler.yml",
        "badge_url": "https://github.com/refined-github/refined-github/workflows/Labeler/badge.svg"
    },
    {
        "id": 10549616,
        "node_id": "MDg6V29ya2Zsb3cxMDU0OTYxNg==",
        "name": "New feature",
        "path": ".github/workflows/new-feature-checker.yml",
        "state": "active",
        "created_at": "2021-06-18T19:18:44.000+09:00",
        "updated_at": "2021-06-18T20:31:13.000+09:00",
        "url": "https://api.github.com/repos/refined-github/refined-github/actions/workflows/10549616",
        "html_url": "https://github.com/refined-github/refined-github/blob/main/.github/workflows/new-feature-checker.yml",
        "badge_url": "https://github.com/refined-github/refined-github/workflows/New%20feature/badge.svg"
    },
    {
        "id": 3562699,
        "node_id": "MDg6V29ya2Zsb3czNTYyNjk5",
        "name": "Release",
        "path": ".github/workflows/release.yml",
        "state": "active",
        "created_at": "2020-11-12T14:56:40.000+09:00",
        "updated_at": "2022-07-07T22:21:40.000+09:00",
        "url": "https://api.github.com/repos/refined-github/refined-github/actions/workflows/3562699",
        "html_url": "https://github.com/refined-github/refined-github/blob/main/.github/workflows/release.yml",
        "badge_url": "https://github.com/refined-github/refined-github/workflows/Release/badge.svg"
    },
    {
        "id": 4737,
        "node_id": "MDg6V29ya2Zsb3c0NzM3",
        "name": "Test",
        "path": ".github/workflows/test.yml",
        "state": "active",
        "created_at": "2019-12-19T10:57:00.000+09:00",
        "updated_at": "2019-12-19T10:57:00.000+09:00",
        "url": "https://api.github.com/repos/refined-github/refined-github/actions/workflows/4737",
        "html_url": "https://github.com/refined-github/refined-github/blob/main/.github/workflows/test.yml",
        "badge_url": "https://github.com/refined-github/refined-github/workflows/Test/badge.svg"
    }
]
```

- API response returns some workflow's `path` value to empty string.
  - The 0th of array which name is "Test" is that.
- The deleted workflow's state should be 'deleted' but 'active' is coming.
  - `.github/workflows/new-feature-checker.yml` is deleted workflow.
```json
"state": {
  "type": "string",
  "enum": [
    "active",
    "deleted",
    "disabled_fork",
    "disabled_inactivity",
    "disabled_manually"
  ],
  "examples": [
    "active"
  ]
},
```

I've created GitHub Issue ticket.